Source/Core/InputCommon/ControllerInterface/DInput/DInputKeyboardMouse.h

```diff
@@ -106,11 +116,36 @@ class KeyboardMouse : public Core::Device
 
 private:
   void UpdateCursorInput();
+  void GenerateOctagonPoints(std::array<POINT, 8> octagon_points) const ;
+  bool IsPointInsideOctagon(const POINT& mouse_point, const std::array<POINT, 8> octagon_points) const;
```

Comment:
```suggestion
  bool IsPointInsideOctagon(const POINT& mouse_point, const std::array<POINT, 8>& octagon_points) const;
```

Ditto for the other ones below, otherwise this creates a copy of the passed in array

Reply:
Yup. Ty. I missed that when I was refactoring.